src/builder/components/question/card/survey-question-card.component.less

```diff
@@ -0,0 +1,39 @@
+.question-card {
+  border: 1px solid #eee;
+  margin-bottom: 20px;
+  display: flex;
+  align-self: 'stretch';
+
+  .question {
+    flex-grow: 1;
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    justify-content: space-between;
+
+    .question-highlight {
```

Comment:
This can be `&-highlight` (same for the classes below), `&` just references the parent selector, `.question` in this case

Reply:
Oh that's pretty cool. TIL. Finally a use for LESS